fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/AbstractJmsIT.java

```diff
@@ -119,61 +125,68 @@ public void testIngestion() throws RepositoryException {
 
         LOGGER.debug("Expecting a {} event", RESOURCE_CREATION.getType());
 
-        final Session session = repository.login();
+        final FedoraSession session = repository.login();
+        session.addSessionData(BASE_URL, TEST_BASE_URL);
```

Comment:
In many of the tests in this class, it is not obvious to me why the addition of session-data is required. Do you have any insights?